coins/src/adapters/yield/arbera/arbera.ts

```diff
@@ -0,0 +1,159 @@
+import { ChainApi } from "@defillama/sdk";
+import axios from "axios";
+import { formatEther } from "ethers";
+
+import { addToDBWritesList } from "../../utils/database";
+import { Write } from "../../utils/dbInterfaces";
+import indexlensAbi from "./indexlens.abi";
+
+const CHAIN_NAME = "berachain";
+const CHAIN_ID = 80094; // berachain
+
+const api = new ChainApi({ chain: CHAIN_NAME });
+
+// ERC20 contracts
+const THREE_BC = "0x467aA1bFa3dCC714f7c16B3D779200431f6A833B";
+const ARBERO = "0xFa7767BBB3d832217ABaa86e5f2654429B3BF29F";
+const USDC_E = "0x549943e04f40284185054145c6E4e9568C1D3241";
+const BRARBERO = "0x0C1f965EB5221B8Daca960dAc1cCfdA5a97B7Dd7";
+const BRLBGT = "0x883899D0111d69f85Fdfd19e4B89E613F231B781";
+const STLBGT = "0xFace73a169e2CA2934036C8Af9f464b5De9eF0ca";

```

Comment:
We have just added pricing for stLBGT and therefore no longer need custom pricing